src/main/java/gov/nysenate/ams/controller/LibraryInfoController.java

```diff
@@ -0,0 +1,54 @@
+package gov.nysenate.ams.controller;
+
+import gov.nysenate.ams.client.response.LibraryInfoResponse;
+import gov.nysenate.ams.filter.ApiFilter;
+import gov.nysenate.ams.provider.AmsNativeProvider;
+import gov.nysenate.ams.util.Application;
+import org.apache.log4j.Logger;
+
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+/**
+ * Servlet to handle library info requests.
+ */
+public class LibraryInfoController extends BaseApiController
+{
+    private Logger logger = Logger.getLogger(LibraryInfoController.class);
+    private AmsNativeProvider amsNativeProvider;
+
+    @Override
+    public void init(ServletConfig config) throws ServletException
+    {
+        this.amsNativeProvider = Application.getAmsNativeProvider();
+        logger.debug("Initialized LibraryInfoController.");
+    }
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException
+    {
+        processRequest(request, response);
+    }
+
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException
```

Comment:
initially this was only going to support `GET` but the `BaseApiController` requires both `doGet` and `doPost`, so I had them both do the same thing. Open to suggestions if there is a preferred approach